packages/governance/README.md

```diff
@@ -0,0 +1,121 @@
+contract to publish details of how its parameters will be subject to governance.
+
+## ContractGovernor
+
+We want some contracts to be able to make it visible that their internal
+parameters can be controlled by a public process, and allow observers to see who
+has control, and how those changes can happen. To do so, the contract would
+incorporate a ParamManager, which has a public facet that provides access to
+those parameters, and a privateFacet, shared only with a ContractGovernor, which
+can make the Registrar visible, while tightly controlling the process of
+creating new questions and presenting them to the electorate.
+
+The governor starts up the Contract and can see what params are subject to
+governance. It provides a private facet that carries the ability to request
+votes on changing particular parameters. Some day we may figure out how to make
+the process and schedule of selecting parameter changes to vote on also subject
+to governance, but that's too many meta-levels at this point.
+
+The party that has the question-creating facet of the ContractGovernor can
+create a question that asks about changing a particular parameter on the
+contract instance. The registrar creates new questions, and makes a new instance
+of a BallotCounter so everyone can see how ballots will be counted.
+
+Registrars have a public method to get from the ballot handle to a ballot.
+ballots include the ballotSpec, the BallotCounter instance and closingRule. For
+contract governance, the question specifies the governed contract instance, the
+parameter to be changed, and the proposed new value.
+
+This is sufficient for voters and others to verify that the contract is managed
+by the governor, the registrar is the one the governor uses, and a particular
+ballotCounter is in use.
+
+## Scenarios
+
+### Examining a Contract before use
+
+Governed contracts will make their governor and parameters visible, either
+through the terms or the public facet. The governor, in turn, publicly shares
+the registrar, which makes the list of questions visible. The questions show
+their ballotCounters, which makes it possible to tell how the counting will be
+done.
+
+There isn't currently a way to verify the process of creating new questions.
+We'll eventually need to spin a story that will make that more legible.
+Currently, the abilty to create new governance questions is provided as a
+private facet that contains only the method voteOnParamChange(). 
+
+When a prospective user of a contract receives a link to an instance of a
+contract, they can check the terms to see if the contract names a governor.  The
+governor's public facet will also refer to the contract it governs. With the
+instance links you'll eventually be able to retrieve the installation from Zoe
+(https://github.com/Agoric/agoric-sdk/issues/3449), which allows you to examine
+the souce.
+
+The governedContract will provide the registrar, which allows you to check the
```

Comment:
How does a user check the electorate? As far as I can tell, the governedContract gives me an instance for a registrar that may or may not actually be the instance associated with the passed-in `registrarCreatorFacet`. So I could be misled to another instance altogether. Putting that aside, let's say I get the installation for the registrar instance, and that it's for the committee registrar. How do I know who is voting?